exampleSite/content/blog/rich-content.md

```diff
@@ -10,23 +10,24 @@ math: false
 toc: false
 ---
 
-Hugo ships with several [Built-in Shortcodes](https://gohugo.io/content-management/shortcodes/#use-hugo-s-built-in-shortcodes) for rich content, along with a [Privacy Config](https://gohugo.io/about/hugo-and-gdpr/) and a set of Simple Shortcodes that enable static and no-JS versions of various social media embeds.
+Hugo ships with several
+[Built-in Shortcodes](https://gohugo.io/content-management/shortcodes/#use-hugo-s-built-in-shortcodes)
```

Comment:
hmm this new line before a square bracket looks worrying. can we ignore md files?

Reply:
That’s weird I didn’t notice this. Does it affect the render output? I’m not at the computer right now.

Reply:
I haven't checked, lemme pull this locally

Reply:
looks like it doesn't affect the render output since Markdown treats single line breaks as line continuation, but yeah I think let's ignore MD files

Reply:
Ok I'll ignore the entire `content` directory